### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jammm**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,14 +2,14 @@
 
 "Yamata No Orochi" [^1] 
 
-A library loads HIP and CUDA driver apis dynamically which allows a user to switch APIs in runtime so you don't need to maintain two similar implementation. 
+Orochi is a library that allows you to load HIP and CUDA API's dynamically which allows the user to switch APIs at runtime so you don't need to compile two separate implementations for each API. Unlike HIP which uses hipamd or CUDA at compile-time, Orochi will dynamically load the corresponding HIP/CUDA shared libraries depending on your platform. In other words, it combines the functionality offered byh HIPEW and CUEW into a single library.
```
Comment (mehmetoguzderin) [inline README.md:5]: API's (genitive) should be APIs (plura)

Comment (mehmetoguzderin) [inline README.md:5]: last byh should be by

Comment (mehmetoguzderin) [reply README.md:5]: -

Comment (jammm) [reply README.md:5]: Oops! thanks for catching that.

Comment (mehmetoguzderin) [reply README.md:5]: -

**Review by mehmetoguzderin — CHANGES_REQUESTED** [review]

**Comment by mehmetoguzderin** [comment]
Forgot to send as review, sorry for the duplicate dots!

**Review by mehmetoguzderin — APPROVED** [review]
LGTM for the raised concerns!

**Comment by jammm** [comment]
> Forgot to send as review, sorry for the duplicate dots!

No worries ^^

**Pull request merged**